lib/rules/space-around-operator.js

```diff
@@ -141,6 +141,18 @@ var isImport = function (operator, parent) {
 };
 
 /**
+ * Determine if operator is part an ident
+ *
+ * @param {string} operator - The operator
+ * @param {Object} next - The next node
+ * @param {Object} previous - The previous node
+ * @returns {bool} true / false
+ */
+var isPartialIdent = function (operator, next, previous) {
+  return next.is('interpolation') && previous.is('interpolation');
```

Comment:
Are we 100% sure that next and previous will be defined, we've seen a lot of errors of the type `is of undefined` before.